catalog/registry/catalog-registry-cswinputtransformer/src/main/java/ddf/catalog/registry/converter/RegistryPackageConverter.java

```diff
@@ -0,0 +1,667 @@
+                    }
+                }
+            }
+
+        }
+    }
+
+    private static Map<String, SlotType1> getSlotMap(List<SlotType1> slots) {
+        Map<String, SlotType1> slotMap = new HashMap<>();
+
+        for (SlotType1 slot : slots) {
+            slotMap.put(slot.getName(), slot);
+        }
+        return slotMap;
+    }
+
+    private static Map<String, List<SlotType1>> getSlotMapWithMultipleValues(
+            List<SlotType1> slots) {
+        Map<String, List<SlotType1>> slotMap = new HashMap<>();
+
+        for (SlotType1 slot : slots) {
+            if (!slotMap.containsKey(slot.getName())) {
+                List<SlotType1> slotList = new ArrayList<>();
+                slotList.add(slot);
+
+                slotMap.put(slot.getName(), slotList);
+            } else {
+                slotMap.get(slot.getName())
+                        .add(slot);
+            }
+        }
+        return slotMap;
+    }
+
+    private static String getStringFromIST(InternationalStringType internationalString) {
+        String stringValue = "";
+        List<LocalizedStringType> localizedStrings = internationalString.getLocalizedString();
+        if (CollectionUtils.isNotEmpty(localizedStrings)) {
+            LocalizedStringType localizedString = localizedStrings.get(0);
+            if (localizedString != null) {
+                stringValue = localizedString.getValue();
+            }
+        }
+
+        return stringValue;
+    }
+
+    private static void unsetMetacardAttribute(String metacardAttribute, RegistryMetacardImpl metacard) {
+        if (StringUtils.isNotBlank(metacardAttribute)) {
+            metacard.setAttribute(metacardAttribute, null);
+        }
+    }
+
+    private static void validateOrganization(OrganizationType organization)
+            throws RegistryConversionException {
+        if (StringUtils.isBlank(organization.getId())) {
+            String message =
+                    "Error converting registry object to metacard. Organization must have an ID set.";
+            throw new RegistryConversionException(message);
+        }
+    }
+
+    private static void validatePerson(PersonType person) throws RegistryConversionException {
+        if (StringUtils.isBlank(person.getId())) {
+            String message =
+                    "Error converting registry object to metacard. Person must have an ID set.";
+            throw new RegistryConversionException(message);
+        }
+    }
+
+    private static void validateRegistryObjectType(RegistryObjectType registryObject)
+            throws RegistryConversionException {
+
+        if (StringUtils.isBlank(registryObject.getId())) {
+            String message =
+                    "Error converting registry object to metacard. Registry Object ID must be set.";
+            throw new RegistryConversionException(message);
+        }
+
+        String type = registryObject.getObjectType();
+
+        if (!REGISTRY_OBJECT_TYPE.equals(type)) {
```

Comment:
I don't think we need to check this anymore. We can just set the content-type to 'registry.'+objecttype